obp-api/src/main/scala/code/api/v4_0_0/OBPAPI4_0_0.scala

```diff
@@ -36,403 +36,44 @@ import code.api.v2_1_0.APIMethods210
-                          Implementations3_1_0.getMessageDocsSwagger ::
-                          Implementations3_1_0.updateCustomerEmail ::
-                          Implementations3_1_0.updateCustomerMobileNumber ::
-                          Implementations3_1_0.updateAccount :: 
-                          Implementations3_1_0.addCardForBank :: 
-                          Implementations3_1_0.deleteCardForBank :: 
-                          Implementations3_1_0.getCardForBank :: 
-                          Implementations3_1_0.getCardsForBank :: 
-                          Implementations3_1_0.updatedCardForBank :: 
-                          Implementations3_1_0.updateCustomerIdentity ::
-                          Implementations3_1_0.updateCustomerBranch ::
-                          Implementations3_1_0.updateCustomerCreditLimit ::
-                          Implementations3_1_0.updateCustomerCreditRatingAndSource ::
-                          Implementations3_1_0.updateCustomerData ::
-                          Implementations3_1_0.getMethodRoutings ::
-                          Implementations3_1_0.createMethodRouting ::
-                          Implementations3_1_0.updateMethodRouting ::
-                          Implementations3_1_0.deleteMethodRouting ::
-                          Implementations3_1_0.updateCustomerNumber ::
-                          Implementations3_1_0.createAccount ::
-                          Implementations3_1_0.getPrivateAccountByIdFull ::
-                          Implementations3_1_0.saveHistoricalTransaction ::
-                          Implementations3_1_0.createCardAttribute ::
-                          Implementations3_1_0.updateCardAttribute ::
-                          Implementations3_1_0.getWebUiProps ::
-                          Implementations3_1_0.createWebUiProps ::
-                          Implementations3_1_0.deleteWebUiProps ::
-                          Implementations3_1_0.getBankAccountsBalances ::
-                          Nil
-  
-  
-  // Possible Endpoints from 4.0.0
-  val endpointsOf4_0_0 = 
-    Implementations4_0_0.getBanks :: 
-    Implementations4_0_0.createTransactionRequest :: 
-    Implementations4_0_0.answerTransactionRequestChallenge ::
-    Implementations4_0_0.getDynamicEntities ::
-    Implementations4_0_0.createDynamicEntity ::
-    Implementations4_0_0.updateDynamicEntity ::
-    Implementations4_0_0.deleteDynamicEntity ::
-    Implementations4_0_0.genericEndpoint ::
-    Implementations4_0_0.resetPasswordUrl ::
-    Implementations4_0_0.root ::
-    Implementations4_0_0.getEntitlements ::
-    Nil
+  // Possible Endpoints from 4.0.0, exclude one endpoint use - method,exclude multiple endpoints use -- method,
+  // e.g getEndpoints(Implementations4_0_0) -- List(Implementations4_0_0.genericEndpoint, Implementations4_0_0.root)
+  val endpointsOf4_0_0 = getEndpoints(Implementations4_0_0) - Implementations4_0_0.genericEndpoint
```

Comment:
Here collect all endpoints, and exclude one endpoint, if you want to exclude multiple endpoints, use -- as the comment describes.